entity-table/index.js

```diff
@@ -287,7 +392,31 @@ module.exports = {
 								done(err);
 							});
 						};
-					}), callback);
+					}), async (err, results) => {
+						if (err) {
+							callback(err);
+						} else {
+							await new Promise((resolve, reject) => {
+								async.parallelLimit(oldS3Files.map(async (file) => {
```

Comment:
I think this one will start all the promises at once.  I think it should return an async closure